src/helpers/advancedTypes.ts

```diff
@@ -0,0 +1,15 @@
+import { PropertyDeclaration } from "lit";
+
+/** Can be used to type a TypeScript enum */
+export type Enum = Readonly<{ [key: number]: string | number }>;
+
+/** A wildcard value from an enum */
```

Comment:
- wildcard means a value that matches any value, it does not mean choosing one value from the set

Reply:
New proposed JSDoc comment

![image](https://github.com/user-attachments/assets/0b06db6c-0796-4c95-829d-faed6170cac9)